provider_config.go

```diff
@@ -0,0 +1,56 @@
+package jwtauth
+
+import (
+	"fmt"
+)
+
+// Provider-specific configuration interfaces
+// All providers must implement the CustomProvider interface, and may implement
+// others as needed.
+
+// ProviderMap is used to map a provider name to its provider type
+var ProviderMap = map[string]CustomProvider{
+	"empty": &EmptyProvider{},
```

Comment:
If this is just a placeholder, you might want to add a TODO comment to remove it.

Reply:
Done in 2e6ce6a